corehq/apps/api/odata/tests/test_feed.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.test import TestCase
+from django.urls import reverse
+
+from elasticsearch.exceptions import ConnectionError
+
+from corehq.apps.accounting.models import (
+    BillingAccount,
+    DefaultProductPlan,
+    SoftwarePlanEdition,
+    Subscription,
+    SubscriptionAdjustment,
+)
+from corehq.apps.api.odata.tests.utils import OdataTestMixin
+from corehq.apps.api.resources.v0_5 import ODataCommCareCaseResource
+from corehq.apps.domain.models import Domain
+from corehq.elastic import get_es_new
+from corehq.pillows.mappings.case_mapping import CASE_INDEX_INFO
+from corehq.util.elastic import ensure_index_deleted
+from corehq.util.test_utils import flag_enabled, trap_extra_setup
+from pillowtop.es_utils import initialize_index_and_mapping
+
+
+class TestOdataFeed(OdataTestMixin, TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestOdataFeed, cls).setUpClass()
+        cls.account, _ = BillingAccount.get_or_create_account_by_domain(cls.domain.name, created_by='')
+        plan_version = DefaultProductPlan.get_default_plan_version(SoftwarePlanEdition.STANDARD)
+        cls.subscription = Subscription.new_domain_subscription(cls.account, cls.domain.name, plan_version)
+
+    @classmethod
+    def tearDownClass(cls):
+        SubscriptionAdjustment.objects.all().delete()
+        cls.subscription.delete()
+        cls.account.delete()
+        super(TestOdataFeed, cls).tearDownClass()
+
+    def test_no_credentials(self):
```

Comment:
are these tests repeated from the base class? or subtly different?

Reply:
These are subtly different, which is an argument against including the test cases themselves in the mixin.